### DISCUSSION TITLE
Big number in query for skip OR take

### DISCUSSION THREAD

**Discussion by floralbpatria** [prisma/prisma#13791] [Q&A]
![Screenshot 2022-06-13 at 12 44 49](https://user-images.githubusercontent.com/104969830/173337284-b91244ee-b035-41d0-920b-765181df8fa6.png)

Trying to test my api with different values found that Prisma Client is not working anymore after throwing the error. We need to restart the client in order for other queries to work.

prisma.user.findMany({ skip: 999999999999999999999, take: 10 }) //After making this query
prisma.user.findMany({ skip: 0, take: 10 }) //This query or any other interaction with the database is throwing the same error.

clientVersion: '3.14.0'

**Comment by austincrim** [comment]
Hey @floralbpatria 👋,

This looks like a bug. Could you open a [bug report](https://github.com/prisma/prisma/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml) with more details about your setup?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
